README.md

```diff
@@ -16,15 +16,19 @@ Azure Data Studio is a data management tool that enables you to work with SQL Se
 |Linux    |.tar.gz          |[64 bit][linux-zip] |
 |         |.deb             |[64 bit][linux-deb] |
 |         |.rpm             |[64 bit][linux-rpm] |
+|Mac      |.zip             |[Universal][osx-universal] |
 |Mac      |.zip             |[Intel Chip][osx-zip] |
-
-[win-user]: https://go.microsoft.com/fwlink/?linkid=2204567
-[win-system]: https://go.microsoft.com/fwlink/?linkid=2204568
-[win-zip]: https://go.microsoft.com/fwlink/?linkid=2204772
-[osx-zip]: https://go.microsoft.com/fwlink/?linkid=2204569
-[linux-zip]: https://go.microsoft.com/fwlink/?linkid=2204773
-[linux-rpm]: https://go.microsoft.com/fwlink/?linkid=2204774
-[linux-deb]: https://go.microsoft.com/fwlink/?linkid=2204570
+|Mac      |.zip             |[Apple Silicon][osx-arm64] |
```

Comment:
please follow the style in the insiders section, there should be just one row for Mac.
![image](https://user-images.githubusercontent.com/13777222/202044722-02f346c5-77ad-4b74-ab28-fadb16378a41.png)

Reply:
`&emsp;` here means four spaces

Reply:
If you look closely you will find the other platforms do have different type of packages, and Mac has just one `.zip`, this is similar to how VSCode organize their downloads.

![image](https://user-images.githubusercontent.com/13777222/202074637-f4730a6d-1eb5-4c69-abdf-55286e56f7c9.png)

Reply:
I've made the change directly in your branch.